### PR TITLE
Implemented Changes on Call

### PR DESCRIPTION
**Pull request by krepinnegga**
-Fixed issue with year of experience
-Added driver licencse view function 
-removed unused code

### CODE REVIEW

File: src/pages/SignIn/DriverLogin.jsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ const DriverLogin = () => {
           setLoading(true);
           const response = await apiDriverLogin(values);
           const userData = response.data;
+          console.log(userData)
```
Comment (ayequill) [inline src/pages/SignIn/DriverLogin.jsx:44]: Please you forgot this @krepinnegga

File: src/pages/SignIn/GetToKnow.jsx [hunk 2]
```diff
@@ -73,27 +82,22 @@ const GetToKnow = () => {
...
-      //   email: values?.email,
-      //   phoneNumber: driverNumber,
-      //   licenseFront: values?.licenseFront?.name,
-      //   licenseBack: values?.licenseBack?.name,
-      // };
-      //const apiMarketingRes = await apiRegisterDriver(data);
+    
       const response = await apiPostDriver(bodyFormData);
-      console.log("this is the response =>", response)
+      console.log(response)
```
Comment (ayequill) [inline src/pages/SignIn/GetToKnow.jsx:89]: And this as well @krepinnegga

File: src/pages/SignIn/DriverLogin.jsx (hunk 1 above)
Comment (krepinnegga) [reply src/pages/SignIn/DriverLogin.jsx:44]: got it.

File: src/pages/SignIn/GetToKnow.jsx (hunk 2 above)
Comment (krepinnegga) [reply src/pages/SignIn/GetToKnow.jsx:89]: got it

**Pull request merged**